TIBERIANDAWN/RulesIniSectionBuilder.h

```diff
@@ -126,22 +126,64 @@ class RulesIniSectionBuilder
 		return RulesIniSection::BuildSection(MOD_RULES_SECTION_NAME)
 			.WithDefaultType(UNSIGNED_INT_RULE)
 			.WithRules([](IRulesIniSection& s) {
-				s << NEW_INFANTRY_RULE << STRING_RULE
-				  << s.BuildRule(NEW_INFANTRY_COUNT_RULE)
-					  .WithMax(CHAR_MAX)
-				  << s.BuildRule(INFANTRY_COUNT_RULE)
-					  .WithDefault(INFANTRY_COUNT)
-					  .WithMin(INFANTRY_FIRST)
-					  .WithMax(CHAR_MAX)
-				  
-				  << NEW_UNITS_RULE << STRING_RULE
-				  << s.BuildRule(NEW_UNIT_COUNT_RULE)
-					  .WithMax(CHAR_MAX)
-				  << s.BuildRule(UNIT_COUNT_RULE)
-					  .WithDefault(UNIT_COUNT)
-					  .WithMin(UNIT_FIRST)
-					  .WithMax(CHAR_MAX);
-			});
+			s << NEW_AIRCRAFT_RULE << STRING_RULE
```

Comment:
We should fix the formatting here to match the existing style - easier to read

Reply:
Yeah Visual Studio insisted on jacking up whitespace any time I cut and pasted. I couldn't quite get it to align properly, so I left it as is.